tests/data/episode_season.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "episode_season",
+    "episodes": [
+        {
+                "title": "Episode",
+                "description": "",
+                "published": 0,
+                "guid": "http://example.org/example.mp3",
+                "link": "",
+                "total_time": 0,
+                "payment_url": null,
+                "season": 1,
+                "enclosures": [
+                    {
+                        "file_size": -1,
+                        "url": "http://example.org/example.mp3",
+                        "mime_type": "application/octet-stream"
+                    }
+                ]
+            }
+      ]
+  }
+  
```

Comment:
Please configure your editor to add a trailing newline to the end of a file.

Reply:
fixed